build.html

```diff
@@ -41,16 +41,9 @@
 <!-- END OF HEADER -->
 <!-- MENU OVERLAY -->
 <div
-  class="fl-menu fl-app-menu {{#if location}}fl-menu-default-left{{else}}fl-menu-default{{/if}} {{#if location}}{{#if canGoBack}}hasGoBack{{/if}}{{/if}}">
+  class="fl-menu fl-app-menu {{#if location}}fl-menu-default-left{{else}}fl-menu-default{{/if}} {{#if location}}{{#if canGoBack}}hasGoBack{{/if}}{{/if}} hidden">
```

Comment:
https://share.getcloudapp.com/Jru1ye95

See above how the menu transitions in when the menu appears. It fades in and moves downwards slightly. With `hidden` applied and effectively adds `display: none`, I fear that the transition would be lost. Can you confirm that the transition still works?

Reply:
@tonytlwu Yes, you are right the transition is lost now. I'll add a new commit to fix that.

Reply:
@tonytlwu There is a little problem with the transition in this menu.
As I understand from the [description](https://github.com/Fliplet/fliplet-menu-default#fliplet-default-menu---right) when we click JQ automatically adds an active class. Is there any listener that we could use to change that behavior?
Why do we need this. In order to fix [this issue](https://weboo.atlassian.net/jira/software/projects/OD/boards/9?selectedIssue=OD-356), we need to set the main menu container as hidden (e.g `display: none`) but the [listener](https://github.com/Fliplet/fliplet-menu-default/pull/25/files#diff-82df7b14712db47ddbc76d0b8bc9ff3977cc8e1c2217029413661176afc37f97R67) fires after active class was added and animation is done before we even can remove the hidden class.

Reply:
@YaroslavOvdii How is this different from the other menus? I see that in the other menus you were able to use `setTimeout` [to achieve the desired outcome](https://github.com/Fliplet/fliplet-menu-circle/pull/24/files). Is it because they are not using `data-fl-toggle-menu`? If that's the case, you can change this menu to not use `data-fl-toggle-menu` and use something like `nav-toggle` like the other menus do.

Reply:
@tonytlwu Fixed animation